crypto/conf/conf_mod.c

```diff
@@ -522,7 +522,7 @@ int CONF_parse_list(const char *list_, int sep, int nospc,
     lstart = list_;
     for (;;) {
         if (nospc) {
-            while (*lstart && isspace((unsigned char)*lstart))
+            while (*lstart && ossl_isspace((unsigned char)*lstart))
```

Comment:
I also thought of that, but chosen not to comment. Because it's probably not that unreasonable limitation. Some other isXXX could be completely different question. In other words *formally* speaking dependency on locale is justified here, but on *practical* level it probably doesn't matter specifically in isspace case. But since a voice was risen, I'd say let's honour the formal approach, not practical.